src/acceleration_structure.rs

```diff
@@ -7,6 +7,16 @@
 
 use super::*;
 
+bitflags::bitflags! {
+    #[derive(Copy, Clone, Debug, Default, Hash, PartialEq, Eq, PartialOrd, Ord)]
+    pub struct MTLAccelerationStructureUsage: u32 {
```

Comment:
This is a `NSUInteger` internally.

```suggestion
    pub struct MTLAccelerationStructureUsage: usize {
```